### DISCUSSION TITLE
How to setup user created - assign role workflow

### DISCUSSION THREAD

**Discussion by jaypeemayo** [OrchardCMS/OrchardCore#10332] [General]
I have this workflow
![image](https://user-images.githubusercontent.com/21273774/134766292-b00942b4-309e-4e50-81ef-d0eb241c255f.png)

I have this Task, I want the created user to be assigned to the role "Creator"
![image](https://user-images.githubusercontent.com/21273774/134766270-21a1cd3c-19c6-46a7-9f7f-dae49590ee8e.png)

Not sure why this is not working.

**Comment by jaypeemayo** [comment]
solved it by using

{{Workflow.Input["User"]}}

**Reply by jtkech** [reply]
Cool!

Funny, when I saw this it was my first thinking.
But didn't take the time to try it ;)
Thanks for confirming!

**Comment by JoshTango** [comment]
Ok yes I can confirm this works.  thanks so much I was looking forever for this answer.
